src/window.rs

```diff
@@ -67,8 +61,10 @@ mod imp {
                 None,
             );
 
-            klass.install_action("sidebar.begin-chats-search", None, move |widget, _, _| {
-                widget.begin_chats_search();
+            klass.install_action("sidebar.begin-chats-search", None, |widget, _, _| {
+                Self::from_instance(widget)
```

Comment:
```suggestion
                self
```

Reply:
Ah, nevermind, I didn't notice they were in clone! macros. I shouldn't review code at almost 3 AM :p